### PR TITLE
Changes to make prod match FTTB

### PR DESCRIPTION
**Pull request by cidlockie**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/features/claimant-response/views/accept-payment-method.njk [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {% from "form.njk" import saveAndContinueButton, csrfProtection, radioGroup, radioGroupWithHintPanel, errorSummary %}
 {% from "./macro/repaymentPlanBlock.njk" import repaymentPlanBlock %}
-{% set heading = 'How they want to pay' %}
+{% set heading = 'Accept or reject their repayment plan' %}
```
[the remark this answers is not in the document]
Comment (sabahirfan) [reply src/main/features/claimant-response/views/accept-payment-method.njk]: https://github.com/hmcts/cmc-citizen-frontend/blob/46af54f30aae7270e35d4d3a2e60b41b0829bf3f/src/test/features/claimant-response/routes/accept-payment-method.ts#L68

[the remark this answers is not in the document]
Comment (sabahirfan) [reply src/main/features/claimant-response/views/accept-payment-method.njk]: https://github.com/hmcts/cmc-citizen-frontend/blob/46af54f30aae7270e35d4d3a2e60b41b0829bf3f/src/test/features/claimant-response/routes/accept-payment-method.ts#L140

[the remark this answers is not in the document]
Comment (sabahirfan) [reply src/main/features/claimant-response/views/accept-payment-method.njk]: please change on these two lines

[the remark this answers is not in the document]
Comment (cidlockie) [reply src/main/features/claimant-response/views/accept-payment-method.njk]: Is it possible to change the heading on the page without changing the route test? I just want the heading to read 'Accept or reject their repayment plan' without breaking everything.

[the remark this answers is not in the document]
Comment (sabahirfan) [reply src/main/features/claimant-response/views/accept-payment-method.njk]: no. route tests are verifying the heading

[the remark this answers is not in the document]
Comment (sabahirfan) [reply src/main/features/claimant-response/views/accept-payment-method.njk]: Thanks @cidlockie

**Comment by pragnesh-patel** [comment]
:-)

**Pull request merged**